### PR TITLE
chore(repo): update high/critical dependabot alerts

### PR DESCRIPTION
**Pull request by MarkMcCulloh**
As part of this update, removed all usages of jest and ts-node.

Also put in a fix for https://github.com/winglang/wing/issues/4128 even though it was closed as stale.

*By submitting this pull request, I confirm that my contribution is made under the terms of the [Wing Cloud Contribution License](https://github.com/winglang/wing/blob/main/CONTRIBUTION_LICENSE.md)*.

### CODE REVIEW

File: apps/jsii-docgen/test/docgen/view/documentation.test.ts [hunk 1]
```diff
@@ -1,8 +1,6 @@
 import * as path from "path";
 import { Documentation, Language } from "../../../src";
 
-jest.setTimeout(30 * 1000);
```
Comment (staycoolcall911) [inline apps/jsii-docgen/test/docgen/view/documentation.test.ts:4]: Is the timeout not needed for this test?

**Review by staycoolcall911 — APPROVED** [review]

File: apps/jsii-docgen/test/docgen/view/documentation.test.ts (hunk 1 above)
Comment (skyrpex) [reply apps/jsii-docgen/test/docgen/view/documentation.test.ts:4]: I believe it's now defined in the config file https://github.com/winglang/wing/pull/5781/files#diff-8384bc5ec390c8c8ac0c0c720db41b82ed8bb903afaccc803dd6cf660efa5315R7

File: tools/bump-pack/src/bundle-link.ts [hunk 2]
```diff
@@ -94,6 +94,14 @@ export async function linkBundledTransitiveDeps(
       if (!(await fs.pathExists(dep.path))) {
         throw new Error(`pnpm dependency path not found: ${dep.path}`);
       }
+
+      console.log(destModule);
```
Comment (skyrpex) [inline tools/bump-pack/src/bundle-link.ts:98]: Is this intended?

**Review by skyrpex — APPROVED** [review]
LGTM.

**Pull request merged**